### DISCUSSION TITLE
Using as embedded html render

### DISCUSSION THREAD

**Discussion by VISTALL** [LoboEvolution/LoboEvolution#65] [Q&A] [answered]
Hello. I'm developer of java IDE, and in some use lobo1 as html render. I wanna upgrade library to new version, but have many problems:

* SQLite dependency, and using settings from user home (which is restricted to embedded  html render)
* There no override logic for services. For example I want override initializing LAFSettings, and other classes with SQL calls.

It will be nice have ApplicationContext with service factories. If not defined - used default implementation. Also it will provide options for override some services for another cases.

It will be nice - having depencency, like loboevolution-sqlite - and this library provide settings via sqllite

Thanks

**Comment by oswetto** [comment]
Hello @VISTALL loboevolution is was rewrite from zero and at moment don't work as embedded. Any help is welcome!

**Comment by oswetto** [comment, marked as answer]
@VISTALL you can retest to embedded loboevolution as html render, sql dependencies have been rewrite

**Reply by VISTALL** [reply]
Thanks, for notify. I will check in my free time :)
